### PR TITLE
feat: add import select screen

### PR DESCRIPTION
**Pull request by MuckT**
### Description

Adds an import select screen when the Statsig feature gate `show_cloud_account_backup_restore` is set to true. By default, when the flag is false, the old restore flow should be served.

<p align="left">

</p>

### Test plan

- [x] Tested Locally on iOS and Android (feature flags were tested by manually hardcoding - no address to target)
- [x] Unit tests added

### Related issues

- Fixes #ACT-874

### Backwards compatibility

Yes

### CODE REVIEW

File: src/importSelect/ImportSelect.tsx [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+  const { t } = useTranslation()
+  const onboardingProps = useTypedSelector(onboardingPropsSelector)
+
+  const handleNavigateBack = () => {
+    dispatch(cancelCreateOrRestoreAccount())
+    ValoraAnalytics.track(OnboardingEvents.restore_account_cancel)
+    navigateClearingStack(Screens.Welcome)
+  }
+
+  useLayoutEffect(() => {
```
Comment (MuckT) [inline src/importSelect/ImportSelect.tsx:73]: If we want a header on this screen later we can use the changes that were removed in 3b181ea552784ee0f4244c760167311037cefe20.

File: locales/base/translation.json [hunk 2]
```diff
@@ -873,6 +873,19 @@
       "title": "Balance\n<0></0>"
     }
   },
+  "importSelect": {
+    "header": "Restore Method",
```
Comment (jh2oman) [inline locales/base/translation.json:877]: not sure where this comes from

File: src/importSelect/ImportSelect.tsx [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+                onboardingProps: {
+                  ...onboardingProps,
+                  chosenRestoreType: ChosenRestoreType.Mnemonic,
+                },
+              })
+            }}
+            testID="ImportSelect/Mnemonic"
+          />
+        </View>
+      </ScrollView>
```
Comment (jh2oman) [inline src/importSelect/ImportSelect.tsx:127]: You are missing the 'Create a new wallet' button https://www.figma.com/file/Qj4kphntYzlYwex79XTcEj/Wallet-Recovery-1.5?node-id=3604%3A33922&mode=dev

**Review by jh2oman — CHANGES_REQUESTED** [review]

File: src/importSelect/ImportSelect.tsx (hunk 3 above)
Comment (MuckT) [reply src/importSelect/ImportSelect.tsx:127]: Part of the AC for the ticket specifies it should not be included: [ACT-874](https://linear.app/valora/issue/ACT-874/restore-options-screen-emailphone-or-manual).

**Review by jh2oman — APPROVED** [review]

File: src/icons/CloudCheck.tsx [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+import colors from 'src/styles/colors'
+
+interface Props {
+  color?: string
+  width?: number
+  height?: number
+}
+
+const CloudCheck = ({ color = colors.primary, width = 20, height = 14 }: Props) => (
+  <Svg width={width} height={height} fill="none">
```
[the remark this answers is not in the document]
Comment (MuckT) [reply src/icons/CloudCheck.tsx]: Fixed in 413b581335df37a197143e2bd629f84b9d312670!

File: src/icons/Lock.tsx [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+import colors from 'src/styles/colors'
+
+interface Props {
+  color?: string
+  width?: number
+  height?: number
+}
+
+const Lock = ({ color = colors.primary, width = 20, height = 20 }: Props) => (
+  <Svg width={width} height={height} fill="none">
```
[the remark this answers is not in the document]
Comment (MuckT) [reply src/icons/Lock.tsx]: Fixed in 413b581335df37a197143e2bd629f84b9d312670!

**Pull request merged**